### DISCUSSION TITLE
Using DigitalOcean Spaces in Volo.Abp.BlobStoring.Aws

### DISCUSSION THREAD

**Discussion by ghulamostafa** [abpframework/abp#15277] [Q&A] [answered]
How can I use the package `Volo.Abp.BlobStoring.Aws` for storing media in DigitalOcean spaces? It supports other SDKs but I would like to know if there is a way I can use the from ABP.

**Comment by realLiangshiwei** [comment, marked as answer]
https://github.com/abpframework/abp/issues/15280#issuecomment-1370398230
